include/nlohmann/json.hpp

```diff
@@ -3793,20 +3898,23 @@ class basic_json
             const auto it = find(key);
             if (it != end())
             {
-                return *it;
+                return it->template get<ValueType>();
```

Comment:
Why this change?

Reply:
This code was using the implicit conversion operator.